defaults/main.yml

```diff
@@ -1,6 +1,8 @@
 ---
 spark_version: "1.3.1-bin-hadoop2.6"
 spark_mirror: "http://d3kbcqa49mib13.cloudfront.net"
+spark_install_java: yes      # Required to check if you need to install Java dependency
```

Comment:
Could you also add this to the `README`?

Reply:
done !